### DISCUSSION TITLE
Grafana 8.0.2 error on module.js

### DISCUSSION THREAD

**Discussion by sibberio** [grafana/grafana-json-datasource#129] [Q&A] [answered]
Hello there! Thanks for this beautiful plugin!
I have an issue with the latest version of Grafana (in docker)
This is the error

Error: Fetch error: 404 Not Found Instantiating http://grafanaIP:3000/public/plugins/marcusolsson-json-datasource/module.js Loading plugins/marcusolsson-json-datasource/module

Can you please me how can I solved it?

**Comment by marcusolsson** [comment, marked as answer]
This might be due to permission issues. Could you check and see if [this solution](https://twitter.com/foosel/status/1404097464278339589) works for you as well?

**Reply by sibberio** [reply]
Wow thanks for your very very quick reply! I have the container on Synology NAS too. I can confirm that the command chmod -R o-x for the folder solved the issue
![image](https://user-images.githubusercontent.com/9988958/122182212-5d3cb380-ce8a-11eb-82e4-f4101a89a38e.png)
Here you can see the "before" and "now"

Many many thanks!

**Comment by neerajnagpal0408** [comment]
Hello
I am using Grafana 6.4.4

[root@ip-10-17-86-55 plugins]# grafana-cli -v Grafana cli version 6.4.4 

but after loading the plugin I am getting the same error :   Error: Object prototype may only be an Object or null: undefined Evaluating http://xx.xx.xx.xx:3000/public/plugins/marcusolsson-json-datasource/module.js Loading plugins/marcusolsson-json-datasource/module Data Source Error.

As per (https://github.com/marcusolsson/grafana-json-datasource/discussions/129) I tried to change the permissions using 

[root@ip-10-17-86-55 plugins]# chmod -R o-x marcusolsson-json-datasource/ [root@ip-10-17-86-55 plugins]# pwd /var/lib/grafana/plugins. --- but it did not help, can you please comment..
